js/profile-w3c-common.js

```diff
@@ -14,8 +14,7 @@ require.config({
   },
   deps: [
     "core/jquery-enhanced",
-    "core/respec-ready",
-    "Promise",
+    "core/respec-ready"
```

Comment:
I think we still want to list "Promise" as a dependency, if only to signal that that is its role.